packages/patternfly-4/react-core/src/components/Dropdown/Dropdown.js

```diff
@@ -61,7 +61,7 @@ class Dropdown extends React.Component {
       position,
       ...props
     } = this.props;
-    const id = toggle.props.id || `pf-toggel-id-${currentId++}`;
+    const id = toggle.props.id || `pf-toggle-id-${currentId++}`;
```

Comment:
This component will be  switched to a class once issue  #771 gets resolved.  I thought it would be to much over head to change it and back it since it functions as is.